modules/aws-outposts-ingress-controller-service.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-ingress-controller-service_{context}"]
+= Configuring the default Ingress Controller in an AWS VPC cluster extended into an Outpost
+
+Classic Load Balancers in the AWS VPC cluster can attach to edge compute resources in the Outpost if edge and cloud-based subnets are in the same availability zone.
+As a result, Classic Load Balancers on the VPC cluster might schedule pods on either of these node types.
+
+The default Ingress Controller deployment on the VPC cluster is scheduled by a Classic Load Balancer.
+Because the load balancer can attach to edge compute resources in the Outpost, it might schedule Ingress Controller deployment on edge compute nodes.
```

Comment:
The phrase doesn't seem correct to me. The Class Load Balancer (CLB) can target (load balancer the traffic between) outpost and cloud based instances. The IngressController deployment can be scheduled on outpost and cloud based nodes but this is done by OpenShift, not CLB.